### DISCUSSION TITLE
Using labels to select entities for cards

### DISCUSSION THREAD

**Discussion by jamie-tm** [home-assistant/frontend#20933] [Dashboard]
### The Request
For the cards that take lists of entities (specifically, in my case, the Logbook and Entities cards), I would love a feature to select all relevant entities using a label, rather than selecting them all individually for each card.

Cards with multiple entities support for labels:
- [ ] Calendar
- [ ] Entities (https://github.com/home-assistant/frontend/pull/25438)
- [ ] Entity Filter
- [ ] Glance
- [ ] History Graph
- [x] Logbook 
- [ ] Map
- [ ] Picture Glance
- [ ] Statistics Graph

Or, alternatively, set the list of entities using a template.

### The Use-Case
As part of my security system, I've got many different sensors that can trigger events and that I want displayed on my dashboard. Each of these entities appears on _at least_ two cards, usually three or four.

For example, I use Logbook Card that to (ideally) display _all_ security events throughout the system:
![image](https://github.com/home-assistant/frontend/assets/10095474/17837a68-c732-4c43-b7d8-bd27e80cc428)

However, the sensors in that example are only a small subset of the ones I actually want. That list will be much, much longer by the time I'm done. And, as I said, each those entities all end up being used in a bunch of places. So I have to remember to update _all_ those places whenever I add or update a sensor. Labels have the potential to make that a whole lot easier!

I can already use templates to set up automations (both triggers and actions) using labels. So the only missing piece is updating the dashboards as well.

### Example
Let's say I want to add a new door sensor for the basement bulkhead, which should never usually be opened.

**Without labels**, I need to:
1. Connect and configure the sensor
1. Add it as a new trigger to the "Alarm Triggers: Home" automation
1. Add it as a new trigger to the "Alarm Triggers: Away" automation
1. Add it as a new trigger to the "Alarm Triggers: Night" automation
1. Add it to the "Security Events" Logbook Card on my "Security" dashboard
1. Add it to the "Doors" Entities Card on my "Security" dashboard
1. Add it to my "Basement" dashboard

**With labels**, I could:
1. Connect and configure the sensor
1. Add the "Alarmed: Always" label

With the features currently available, my three automations can use that label (either with a template, or through a template entity such as `binary_sensor.alarmed_always_sensor_tripped`) to automatically pick up the new sensor. That's all (relatively) easy to configure through the UI.

Unfortunately, using built-in features, I would still need to update the three dashboards manually.
But with this feature, I could also have the cards auto-update with the labels. Super fast and easy!

### Workaround
Of course, there is _always_ a workaround in Home Assistant, and this is no exception. There _are_ custom cards I can use to generate cards from templates, but those are 3rd party and a lot more complicated to set up. I'll probably take the time to do it anyway, but a friendly, built-in option would be a **lot** easier. Hence the feature request.

**Comment by gdrapp** [comment]
Love this idea

**Comment by asmueller** [comment]
Exactly what I'm looking for!

**Comment by actstorms** [comment]
+1  Good idea

**Comment by bergethan** [comment]
+1 I definitely need this! Both for Alarmo and for the Entity Filter card in Lovelace!

**Comment by cpocarlos** [comment]
That is what I am looking for

**Comment by givaldolins** [comment]
+1 for this

**Comment by VladDBA** [comment]
+1

**Comment by andytrofimov** [comment]
+1

**Comment by xionc666** [comment]
+1

**Comment by karwosts** [comment]
Pull <https://github.com/home-assistant/frontend/pull/25438> has been pending for this for entities card for a while.

**Comment by karwosts** [comment]
Also I see the original post uses Logbook card as an example; that already has label support.
